CONTRIBUTING.md

```
@@ -1 +1,135 @@
+You'll find code related to `@vahor/n8n-kit-cli` in the [`packages/n8n-cli`](./packages/n8n-cli) directory.
+
+
 `/check-examples` command in PR to check if examples still work.
+
+## Types of Contributions
+
+### Updating generated files
+
+If the [n8n](https://github.com/n8n-io/n8n) repository has been updated with new version of nodes, you'll have to follow the steps below to update the generated files.
+
+1. Update the submodule:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)